[0.057s] Invoking command in '/home/tjacobs/ros2_ws/build/agrobot_msgs': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/tjacobs/ros2_ws/install/my_package:/home/tjacobs/ros2_ws/install/agrobot_pkg:/opt/ros/iron /usr/bin/cmake --build /home/tjacobs/ros2_ws/build/agrobot_msgs -- -j8 -l8
[0.077s] -- Found ament_cmake: 2.0.3 (/opt/ros/iron/share/ament_cmake/cmake)
[0.182s] -- Found rosidl_default_generators: 1.5.0 (/opt/ros/iron/share/rosidl_default_generators/cmake)
[0.183s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.207s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.228s] -- Found rosidl_adapter: 4.0.1 (/opt/ros/iron/share/rosidl_adapter/cmake)
[0.496s] -- Found ament_cmake_ros: 0.11.2 (/opt/ros/iron/share/ament_cmake_ros/cmake)
[0.625s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.798s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.814s] -- Found python_cmake_module: 0.10.2 (/opt/ros/iron/share/python_cmake_module/cmake)
[0.831s] -- Using PYTHON_EXECUTABLE: /usr/bin/python3
[0.831s] -- Using PYTHON_INCLUDE_DIRS: /usr/include/python3.10
[0.831s] -- Using PYTHON_LIBRARIES: /usr/lib/x86_64-linux-gnu/libpython3.10.so
[0.962s] [31mCMake Error at /opt/ros/iron/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:57 (message):
[0.962s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
[0.962s]   agrobot_msgs --arguments-file
[0.962s]   /home/tjacobs/ros2_ws/build/agrobot_msgs/rosidl_adapter__arguments__agrobot_msgs.json
[0.962s]   --output-dir
[0.962s]   /home/tjacobs/ros2_ws/build/agrobot_msgs/rosidl_adapter/agrobot_msgs
[0.963s]   --output-file
[0.963s]   /home/tjacobs/ros2_ws/build/agrobot_msgs/rosidl_adapter/agrobot_msgs.idls)
[0.963s]   returned error code 1:
[0.963s] 
[0.963s]   Error processing 'int64 crop y' of 'agrobot_msgs/VisionPublishClosestCrop':
[0.963s]   'value 'y' can not be converted to type 'int64': must be a valid integer
[0.963s]   value >= -9223372036854775808 and <= 9223372036854775807'
[0.963s] 
[0.963s]   Traceback (most recent call last):
[0.963s] 
[0.963s]     File "/opt/ros/iron/lib/python3.10/site-packages/rosidl_adapter/parser.py", line 753, in parse_primitive_value_string
[0.963s]       value = int(value_string)
[0.963s] 
[0.963s]   ValueError: invalid literal for int() with base 10: 'y'
[0.963s] 
[0.963s]   
[0.963s] 
[0.963s]   During handling of the above exception, another exception occurred:
[0.963s] 
[0.963s]   
[0.963s] 
[0.963s]   Traceback (most recent call last):
[0.963s] 
[0.963s]     File "/opt/ros/iron/lib/python3.10/site-packages/rosidl_adapter/parser.py", line 756, in parse_primitive_value_string
[0.964s]       value = int(value_string, 0)
[0.964s] 
[0.964s]   ValueError: invalid literal for int() with base 0: 'y'
[0.964s] 
[0.964s]   
[0.964s] 
[0.964s]   During handling of the above exception, another exception occurred:
[0.964s] 
[0.964s]   
[0.964s] 
[0.964s]   Traceback (most recent call last):
[0.964s] 
[0.964s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.964s]       return _run_code(code, main_globals, None,
[0.964s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.964s]       exec(code, run_globals)
[0.964s]     File "/opt/ros/iron/lib/python3.10/site-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.964s]       sys.exit(main())
[0.964s]     File "/opt/ros/iron/lib/python3.10/site-packages/rosidl_adapter/main.py", line 53, in main
[0.964s]       abs_idl_file = convert_to_idl(
[0.964s]     File "/opt/ros/iron/lib/python3.10/site-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.965s]       return convert_msg_to_idl(
[0.965s]     File "/opt/ros/iron/lib/python3.10/site-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.965s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.965s]     File "/opt/ros/iron/lib/python3.10/site-packages/rosidl_adapter/parser.py", line 520, in parse_message_string
[0.965s]       fields.append(Field(
[0.965s]     File "/opt/ros/iron/lib/python3.10/site-packages/rosidl_adapter/parser.py", line 362, in __init__
[0.965s]       self.default_value = parse_value_string(
[0.965s]     File "/opt/ros/iron/lib/python3.10/site-packages/rosidl_adapter/parser.py", line 595, in parse_value_string
[0.965s]       return parse_primitive_value_string(type_, value_string)
[0.965s]     File "/opt/ros/iron/lib/python3.10/site-packages/rosidl_adapter/parser.py", line 758, in parse_primitive_value_string
[0.965s]       raise ex
[0.965s] 
[0.965s]   rosidl_adapter.parser.InvalidValue: value 'y' can not be converted to type
[0.965s]   'int64': must be a valid integer value >= -9223372036854775808 and <=
[0.965s]   9223372036854775807
[0.965s] 
[0.965s] Call Stack (most recent call first):
[0.965s]   /opt/ros/iron/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:132 (rosidl_adapt_interfaces)
[0.965s]   CMakeLists.txt:20 (rosidl_generate_interfaces)
[0.965s] 
[0.965s] [0m
[0.965s] -- Configuring incomplete, errors occurred!
[0.965s] See also "/home/tjacobs/ros2_ws/build/agrobot_msgs/CMakeFiles/CMakeOutput.log".
[0.967s] gmake: *** [Makefile:815: cmake_check_build_system] Error 1
[0.969s] Invoked command in '/home/tjacobs/ros2_ws/build/agrobot_msgs' returned '2': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/tjacobs/ros2_ws/install/my_package:/home/tjacobs/ros2_ws/install/agrobot_pkg:/opt/ros/iron /usr/bin/cmake --build /home/tjacobs/ros2_ws/build/agrobot_msgs -- -j8 -l8
